source/docs/blockchain_basic_configuration.md

```diff
@@ -0,0 +1,66 @@
+title: Blockchain configuration
+---
+
+When in development phase, it is necessary to use a local Ethereum node to deploy our contracts to, either using a simulator or client like Geth or Parity.
+
+To set this node, Embark looks at `blockchain.js`, which by default is at `config/blockchain.js`.
+
+It is a complex file, but Embark, when using the `embark new` and `embark demo` command, creates a default `blockchain.js` file for you that contains all the basic parameters for you to start a Geth node or a Ganache simulator.
+
+## Common Parameters
+
+Here are the common parameters:
+
+<pre><code class="javascript">module.exports = {
+  default: {
+    enabled: true,
+    rpcHost: "localhost",
+    rpcPort: 8545,
+    rpcCorsDomain: "auto",
+    wsRPC: true,
+    wsOrigins: "auto",
+    wsHost: "localhost", 
+    wsPort: 8546
+  },
+  development: {
+    ethereumClientName: "geth",
+    ethereumClientBin: "geth",
+    datadir: ".embark/development/datadir",
+    networkType: "custom",
+    networkId: 1337,
+    isDev: true,
+    nodiscover: true,
+    maxpeers: 0,
+    proxy: true,
+    targetGasLimit: 8000000
+  }
+}
+</code></pre>
+
+As with `contracts.js`, this config contains environments that help configuring certain parameters differently depending of the environment. You can read more about [environments here](https://embark.status.im/docs/environments.html).
```

Comment:
Instead of mentioning `contracts.js`, which potentially can be named something totally different if the user wants to, how about we rather refer to the guide on configuring contracts?

Something like:

> Similar to [configuring Smart Contracts](), this configuration file contains [environments]() that help [...]